openlibrary/plugins/upstream/mybooks.py

```diff
@@ -648,10 +654,12 @@ def get_observations(self, limit=RESULTS_PER_PAGE, page=1):
             entry['observations'] = convert_observation_ids(ids)
         return observations
 
-    def _get_work(self, work_key):
+    def _get_work(self, work_key: str) -> "Work":
```

Comment:
```suggestion
    def _get_work(self, work_key: str) -> "Work | None":
```
If an invalid Work ID is passed to this function, it would return `None`.